gamestonk_terminal/main_helper.py

```diff
@@ -564,56 +601,6 @@ def view(other_args: List[str], s_ticker: str, s_interval, df_stock):
         return
 
 
-def export(other_args: List[str], df_stock):
-    parser = argparse.ArgumentParser(
-        add_help=False,
-        prog="export",
-        description="Exports the historical data from this ticker to a file or stdout.",
-    )
-    parser.add_argument(
-        "-f",
-        "--filename",
-        type=str,
-        dest="s_filename",
```

Comment:
Are we eliminating the data export and adding it to individual functions?

Reply:
Yup!